### PR TITLE
Subscribe to projects

### PR DESCRIPTION
**Pull request by KochTobi**
This PR introduces a new feature: It allows users to subscribe to a project to get notifications.

- [x] Add the DS to store new subscriptions (#87)
- [x] Add the use case logic to the domain (#88)
- [x] Add a GUI component to trigger subscriptions (#92)
- [x] Notify the user about success or failure of subscriptions (#96)
- [x] Wire together view and use case (#96)

### CODE REVIEW

**Comment by jenniferboedker** [comment]
I think this is ready to review, right @KochTobi?

File: sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/SubscriptionDataSource.groovy [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package life.qbic.business.project.subscribe
+
+/**
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
+ */
```
Comment (wow-such-code) [inline sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/SubscriptionDataSource.groovy:9]: missing description

**Review by wow-such-code — CHANGES_REQUESTED** [review]
NIT: missing a description, otherwise looking good!

File: sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/SubscriptionDataSource.groovy (hunk 1 above)
Comment (Steffengreiner) [reply sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/SubscriptionDataSource.groovy]: Nice Catch 👁️

**Review by wow-such-code — APPROVED** [review]

**Pull request merged**